### ISSUE TITLE
Inspect responsive view is broken in chrome? 

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#4708]
Am i doing something wrong or is it broken? 
![image](https://user-images.githubusercontent.com/90852205/212537514-4bb6aa5e-f25e-4cad-a126-354079ae008b.png)
![image](https://user-images.githubusercontent.com/90852205/212537520-c7ab37f0-6dc2-443c-b74c-96fee3f95264.png)

**Comment by Jarsen136** [comment]
I seems could not reproduce this case. After reloading the page, is this bug still there? If so, pls provide the link to the gallery item.

**Comment by exezbcz** [comment]
> I seems could not reproduce this case. After reloading the page, is this bug still there? If so, pls provide the link to the gallery item.

yup, reloading fixed it, but once i move with the handles its broken again. Seems like everything from 700px and lower has the layout pushed from right. 
![image](https://user-images.githubusercontent.com/90852205/212538779-5fd64856-f9b2-4313-9df5-1981955b91c0.png)
https://beta.kodadot.xyz/snek/gallery/659233203-5?redesign=true

**Comment by petkdev** [comment]
This issue has been resolved?

**Comment by Jarsen136** [comment]
> This issue has been resolved?

I guess not. If you could reproduce it, go for it!

**Comment by exezbcz** [comment]
I'm not sure whether it's just me, but I haven't come up with a fix yet :D

**Comment by JustLuuuu** [comment]
@exezbcz still not fixed?
